src/components/AssetsTable/index.jsx

```diff
@@ -270,6 +223,67 @@ export class AssetsTable extends React.Component {
     });
   }
 
+  updateStatusAlertFields(assetsStatus) {
+    const assetName = this.state.deletedAsset.display_name;
+    let alertDialog;
+    let alertType;
+
+    switch (assetsStatus.type) {
+      case assetActions.DELETE_ASSET_FAILURE:
+        alertType = 'danger';
+        alertDialog = `Unable to delete ${assetName}.`;
+        break;
+      case assetActions.DELETE_ASSET_SUCCESS:
+        alertType = 'success';
+        alertDialog = `${assetName} has been deleted.`;
+        break;
+      case assetActions.UPLOAD_ASSET_SUCCESS:
+        this.updateUploadSuccessCount();
+        alertType = 'success';
+        alertDialog = `${this.state.uploadSuccessCount} files successfully uploaded.`;
+        break;
+      case assetActions.UPLOADING_ASSETS:
+        this.closeStatusAlert();
+        alertType = 'info';
+        alertDialog = `${assetsStatus.count} files uploading.`;
+        break;
+      case assetActions.UPLOAD_EXCEED_MAX_COUNT_ERROR:
+        alertType = 'danger';
+        alertDialog = `The maximum number of files for an upload is ${assetsStatus.maxFileCount}. No files were not uploaded.`;
```

Comment:
'No files were not uploaded' sounds a little funny. I think you don't need the extra 'not'. What do you think?

Reply:
Ah that must have been carried over, I'll remove the 'not'.